suites/pacific/rgw/tier-1_rgw_multisite_multirealm_upgrade-5-to-latest.yaml

```diff
@@ -0,0 +1,561 @@
+# Polarion ID: CEPH-83574647
+# Objective: Testing Multisite upgrade from RHCS 5 GA to RHCS 5 latest development build.
```

Comment:
Can we update objective accordingly,
i.e: Testing multirealm-Multisite upgrade from RHCS 5 GA to RHCS 5 latest development build.

Reply:
Done